config/initializers/001-redis.rb

```diff
@@ -9,7 +9,11 @@
 # upgrade to Sidekiq 6.1
 Redis.exists_returns_integer = true
 
-if Gem::Version.new(Discourse.redis.info['redis_version']) < Gem::Version.new("6.2.0")
-  STDERR.puts "Discourse requires Redis 6.2.0 or up"
-  exit 1
+begin
+  if Gem::Version.new(Discourse.redis.info['redis_version']) < Gem::Version.new("7.2.0")
+    STDERR.puts "Discourse requires Redis 6.2.0 or up"
+    exit 1
+  end
+rescue Redis::CannotConnectError
+  puts "Couldn't connect to Redis"
```

Comment:
So we don't pollute script output
```suggestion
  STDERR.puts "Couldn't connect to Redis"
```